src/pepr/operator/controllers/network/defaults/default-deny-all.ts

```diff
@@ -0,0 +1,18 @@
+import { kind } from "pepr";
+
+export function defaultDenyAll(namespace: string): kind.NetworkPolicy {
+  return {
+    apiVersion: "networking.k8s.io/v1",
+    kind: "NetworkPolicy",
+    metadata: {
+      name: "uds-default-deny-all",
```

Comment:
This name won't be unique in `namespace`.  Is the assumption at the moment that there will be only one Package per namespace?

Reply:
If I create a second package with:
```
apiVersion: uds.dev/v1alpha1
kind: Package
metadata:
  name: foo
  namespace: grafana
```

and then delete it, the generic netpol originally created by the grafana package will be deleted as well.

Reply:
yeah the choices are create a second ownerref (which would require first doing a `get`) or just using a unique name, def. more performant to just use a unique name instead.